db/schema.rb

```diff
@@ -32,6 +32,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.text "upload_log"
     t.boolean "mapbox", default: false
+    t.string "fileStructure"
     t.index ["course_id"], name: "index_assignments_on_course_id"
   end
```

Comment:
This is a new parameter to store whether the user wants to process the uploaded zip file by files or by directory.